textinput/textinput.go

```diff
@@ -15,6 +16,9 @@ import (
 
 const defaultBlinkSpeed = time.Millisecond * 530
 
+var intRegex = regexp.MustCompile("[0-9]")
```

Comment:
Maybe we could avoid using regexps by relying on `strconv.ParseInt`, `strconv.ParseFloat` and `strconv.ParseBool`?

Reply:
done